src/main/java/seedu/edulink/logic/Messages.java

```diff
@@ -62,12 +64,21 @@ public static String format(Student student) {
      * Formats the {@code grade} for display to the user.
      */
     public static String format(Grade grade) {
+        DecimalFormat df = new DecimalFormat("#.##");
         final StringBuilder builder = new StringBuilder();
         builder.append("Module: ")
             .append(grade.getModule())
             .append("; Score: ")
-            .append(grade.getScore());
+            .append(df.format(grade.getScore()));
```

Comment:
getScore returns the Score object. should use .getScore().getScore() because df.format expects double argument